flow/designs/asap7/ibex/config.mk

```diff
@@ -11,4 +11,6 @@ export CORE_ASPECT_RATIO      = 1
 export CORE_MARGIN            = 2
 export PLACE_DENSITY          = 0.60
 
+export ENABLE_DPO = 0
```

Comment:
Do you intend to fix this later?

Reply:
Yes I will file a Jira for the two test cases which we have bypassed DPO.